transmart-schemas/src/main/resources/db/changelog/changes/20191907081438_biomart_user_views_with_bitset.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <preConditions onFail="WARN">
+        <dbms type="PostgreSQL" />
+        <sqlCheck expectedResult="1">
+            SELECT count(*) FROM pg_extension WHERE extname='pg_bitcount'
+        </sqlCheck>
+    </preConditions>
+    <changeSet author="artur@thehyve.nl" id="20191907081438-0" dbms="PostgreSQL">
+        <comment>
+            Materialized views are not supported yet by liquibase
+        </comment>
+        <sql endDelimiter=";">
+            CREATE MATERIALIZED VIEW biomart_user.study_concept_bitset TABLESPACE pg_default AS
+            SELECT   s.study_id,
+                     o.concept_cd,
+                     Pg_int_to_bit_agg((o.patient_num - patient_num_boundaries.min_patient_num)::INTEGER, patient_num_boundaries.diameter::INTEGER) AS patient_set_bits
+            FROM     biomart_user.patient_num_boundaries,
+                     i2b2demodata.observation_fact o
+            JOIN     i2b2demodata.trial_visit_dimension tv
+            ON       o.trial_visit_num = tv.trial_visit_num
+            JOIN     i2b2demodata.study s
+            ON       s.study_num = tv.study_num
+            WHERE    o.modifier_cd::text = '@'::text
+            GROUP BY s.study_id,
+                     o.concept_cd WITH DATA
+        </sql>
+    </changeSet>
+    <changeSet author="artur@thehyve.nl" id="20191907081438-1" dbms="PostgreSQL">
```

Comment:
These multiple changes belong in a single change set.

Reply:
I think we need 2 changesets:

1. combine 0 & 1
2. others.